manubot/cite/isbn.py

```diff
@@ -0,0 +1,84 @@
+import collections
+import json
+import logging
+import re
+
+
+def get_isbn_citeproc(isbn):
+    return get_isbn_citeproc_citoid(isbn)
+
+
+def get_isbn_citeproc_isbnlib(isbn):
```

Comment:
It looks like this function isn't used currently.  Will it be later?

Reply:
This touches on a larger question: do we retain metadata methods that are not used as the default method for a certain metadata source. I think we should because it allows:

1. us to easily switch to the other source in the future
2. may be helpful for certain users who are using manubot for citation metadata retreival

So I kept this in here because `isbnlib` is now a dependency, and thought it made sense to at least make it's metadata retrieval an option.

Reply:
Your reasoning makes sense to me.  I do think it would improve readability to have a comment that `get_isbn_citeproc_isbnlib` is an alternative ISBN metadata retrieval function that is not currently used in the standard ISBN processing pipeline.

Reply:
By the way, I resolved the other parts of our discussion that seem to be addressed.  This suggestion is the only remaining comment that may be actionable, depending on what you think.

Reply:
How about https://github.com/greenelab/manubot/pull/79/commits/9337ef754c3738a0a40cb452ea653aaca56d8a4f?

Reply:
LGTM